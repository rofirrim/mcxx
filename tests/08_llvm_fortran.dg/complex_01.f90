! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="complex_01.f90.out"
! </testinfo>
PROGRAM MAIN
  IMPLICIT NONE
  COMPLEX :: C, D, E

  C = (4, 3)
  D = (8, 6)

  E = C + D
  PRINT *, E

  E = C - D
  PRINT *, E

  E = D - C
  PRINT *, E

  E = D * C
  PRINT *, E

  E = C * D
  PRINT *, E

  E = C / D
  PRINT *, E

  E = D / E
  PRINT *, E
END PROGRAM MAIN
