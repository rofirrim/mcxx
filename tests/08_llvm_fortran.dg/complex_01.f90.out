 (  12.0000000    ,  9.00000000    )
 ( -4.00000000    , -3.00000000    )
 (  4.00000000    ,  3.00000000    )
 (  14.0000000    ,  48.0000000    )
 (  14.0000000    ,  48.0000000    )
 ( 0.500000000    ,  0.00000000    )
 (  16.0000000    ,  12.0000000    )
